### PR TITLE
Added CRMScript: forceSaveTrace method

### PR DESCRIPTION
**Pull request by hjelms**

### CODE REVIEW

**Review by digitaldiina — COMMENTED** [review]
so.version: 7.0 @hjelms ?

File: docs/en/automation/crmscript/reference/CRMScript.Global.Void.yml [hunk 1]
```diff
@@ -1307,6 +1308,29 @@ items:
     return:
       type: CRMScript.Global.Void
   example: 
+- uid: CRMScript.Global.Void.forceSaveTrace(Bool)
+  commentId: M:CRMScript.Global.Void.forceSaveTrace(Bool)
+  id: 'forceSaveTrace(Bool)'
+  so.intellisense: Void.forceSaveTrace
+  so.version: 7.0
```
Comment (SuperOfficeDevNet) [inline docs/en/automation/crmscript/reference/CRMScript.Global.Void.yml:1315]:
```suggestion
  so.version: 10.2.11
```

**Review by SuperOfficeDevNet — DISMISSED** [review]

**Comment by SuperOfficeDevNet** [comment]
> so.version: 7.0 @hjelms ?

Contacted Sverre directly and confirmed it is for 10.2.11, and updated the version.

**Review by digitaldiina — APPROVED** [review]
Thanks for the new method @hjelms💯

**Pull request merged**